src/scripts/main.js

```diff
@@ -1,8 +1,18 @@
 'use strict';
 
 const people = require('./lib/people');
+const table = document.querySelector('.dashboard');
+const tr = document.createElement('tr');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+for (const elem of people) {
```

Comment:
```suggestion
for (const person of people) {
```
use more specified variable names

Reply:
OK. Thanks a lot!